_posts/acs-aem-commons/features/2014-01-01-simple-sitemap.md

```diff
@@ -56,4 +58,10 @@ Typically, you will also want to use either Resource Resolver Mappings or mod_re
 
 * *changefreq.properties* The set of JCR property names which will contain the change frequency value. Defaults to none.
 
-* *priority.properties* The set of JCR property names which will contain the priority value. Defauls to none.
+* *priority.properties* The set of JCR property names which will contain the priority value. Defaults to none.
+
+#### Since v2.3.0
```

Comment:
I hate to put a version number in here at this point b/c the PR for the changes is still pending.  Is there a different way I should be doing this?